packages/react-components/react-spinner/src/components/Spinner/useSpinnerStyles.ts

```diff
@@ -228,6 +228,10 @@ const useTrackStyles = makeStyles({
       strokeLinecap: 'round',
       transform: 'rotate(-90deg)',
       transformOrigin: '50% 50%',
+      '@media screen and (prefers-reduced-motion: reduce)': {
+        animationDuration: '0',
```

Comment:
My approach to reduced motion would be to remove anything that has actual perceived motion (vs. something like fade in/out).

I'd also be fine with the linked approach, though in this case it would actually cause the `animationend` event to fire in reduced motion, while it wouldn't occur otherwise. Is that something we care about vs. implementing a consistent solution everywhere?

Reply:
> My approach to reduced motion would be to remove anything that has actual perceived motion (vs. something like fade in/out).

👍

> I'd also be fine with the linked approach, though in this case it would actually cause the animationend event to fire in reduced motion, while it wouldn't occur otherwise. Is that something we care about vs. implementing a consistent solution everywhere?

I think you'd need to remove the reduced motion override for `animation-iteration-count` but that potentially causes the `animationiteration` event to fire a LOT if `animation-duration` is something like 0.1s :/

The way you have it is fine by me.

Reply:
yeah, without overriding the iteration count, I'd be worried about flickering on some browsers/OSs :/